scripts/file-entities.php.in

```diff
@@ -226,6 +226,11 @@ function file_entities($work_dir, $trans_dir, $orig_dir, &$entities, $prefix=fal
         // Sort by name
         sort($function_entities);
 
+        $function_entities = array_map(
+            fn ($entity): string => $entity . ";",
+            $function_entities
+        );
+
```

Comment:
Why not just output the trailing ";" in the `fputs()` below this and avoid the extra run through the array?

Reply:
Because I didn't pay enough attention to realize I could do that. :-) I will correct this shortly. :-)